Gemfile.lock

```diff
@@ -91,6 +91,19 @@ GIT
       naught (~> 1.0)
       simple_oauth (~> 0.3.0)
 
+GIT
+  remote: https://github.com/yubuylov/huginn_mysql2_agent.git
+  revision: 51b71bba455d07b97fd0c178f078dd96c51f4adb
+  specs:
+    huginn_mysql2_agent (0.1)
+      huginn_agent (~> 0.2)
+
+PATH
+  remote: ../huginn_mattermost_form
+  specs:
+    huginn_mattermost_form (0.1)
+      huginn_agent
+
```

Comment:
Can you exclude these from this PR?

Reply:
Think I got it.  I had forgotten about this pull request.

Reply:
@paul-sx Sorry for being extremely late to respond. 😣  Just one more favor, could you squash all the commits into one?